neuron/src-bash/neuron-search

```diff
@@ -9,6 +9,8 @@ EXTENSIONS=${4}
 OPENCMD=$(envsubst -no-unset -no-empty <<<${5})
 EXACT=""
 
+: ${NEURON_SEARCH_CONTEXT_ABOVE:=5}
```

Comment:
@srid  Does this need to be able to be disabled btw, back to original "only show the start of the zettel"?